alpaca/alpaca_test.go

```diff
@@ -372,24 +372,45 @@ func (s *AlpacaTestSuite) TestAlpaca() {
 			return &http.Response{
 				Body: genBody(Order{
 					Qty:         por.Qty,
+					Notional:    por.Notional,
 					Side:        por.Side,
 					TimeInForce: por.TimeInForce,
 					Type:        por.Type,
 				}),
 			}, nil
 		}
 
+		qty := decimal.New(1, 0)
 		req := PlaceOrderRequest{
 			AccountID:   "some_id",
-			Qty:         decimal.New(1, 0),
+			Qty:         qty,
 			Side:        Buy,
 			TimeInForce: GTC,
 			Type:        Limit,
 		}
 
 		order, err := PlaceOrder(req)
-		assert.Nil(s.T(), err)
+		assert.NoError(s.T(), err)
+		assert.NotNil(s.T(), order)
+		assert.Equal(s.T(), req.Qty, order.Qty)
+		assert.Zero(s.T(), decimal.Decimal{}, "", req.Notional, order.Notional)
```

Comment:
This assert doesn't check anything. It checks that `decimal.Decimal{}` is zero (always true). The rest of the arguments are just for printing. 
```suggestion
		assert.True(s.T(), req.Notional.IsZero())
		assert.True(s.T(), order.Notional.IsZero())
```